src/index.css

```diff
@@ -31,3 +31,13 @@ code {
 .align-center {
   text-align: center;
 }
+
+.btn-theme {
+  width: 30px;
+  height: 30px;
+
+  border: none;
+  border-radius: 50%;
+
+  background-color: grey;
+}
```

Comment:
Probably better to go in a component style like `Header.css`, but even better could we use PrimeReact components

- Buttons https://www.primefaces.org/primereact/showcase/#/button
- Toogles https://www.primefaces.org/primereact/showcase/#/togglebutton

Reply:
If we are using PrimeReact then I think using switches will look better?

- https://www.primefaces.org/primereact/showcase/#/inputswitch

Reply:
Great 👍